doc/api/errors.md

```diff
@@ -1720,15 +1703,6 @@ A `Transform` stream finished with data still in the write buffer.
 
 The initialization of a TTY failed due to a system error.
 
-<a id="ERR_UNCAUGHT_EXCEPTION_CAPTURE_ALREADY_SET"></a>
-### ERR_UNCAUGHT_EXCEPTION_CAPTURE_ALREADY_SET
```

Comment:
We have a section for errors that were once present but are no longer below in this document :)

Reply:
Thanks for the pointer 👍  I'll add those errors to the [`Legacy Node.js Error Codes` section](https://github.com/nodejs/node/blob/49b0f7fa19ce4b8fc3930124e58496894d4ea932/doc/api/errors.md#legacy-nodejs-error-codes). Just to make sure: is it the section you were referring to?

Reply:
Yes, exactly. :)